codepropertygraph/build.sbt

```diff
@@ -47,12 +47,18 @@ Compile / sourceGenerators += Def.task {
     println("regenerating domain classes")
     DomainClassCreator.run((Compile / sourceManaged).value)
 
+// # equivalent of `sourceManaged.in(Compile).value` in build.sbt
+// OUTPUT_DIR = '../../../../target/src_managed/main/io/shiftleft/codepropertygraph/generated'
     // TODO: port python to jpython, scala or java to avoid system call and pass values in/out
     val cmd = "codepropertygraph/codegen/src/main/python/generateJava.py"
     val result = Seq(cmd).!
-    if (result == 0)
-      println(s"successfully called $cmd")
-    else
+    if (result == 0) {
+      val outputRootBF = better.files.File(outputRoot.toPath)
+      val generateJavaTmpOutput = better.files.File("codepropertygraph/target/generateJava")
```

Comment:
what's the reason for changing this path?

Reply:
the build used to not append the scala major version to the built artifact, which was a bit of a lie:
`crossBuild := false`. So I dropped that:
https://github.com/ShiftLeftSecurity/codepropertygraph/pull/334/files#diff-fdc3abdfd754eeb24090dbd90aeec2ceL9
that's why a few paths changed